CONTRIBUTING.md

```diff
@@ -37,29 +36,29 @@ tagged with the **[easy pick]** label could be a good choice for newcomers (and
 we are willing to help you).
 
 We have two kind of bugs: **[critical]** and **[bug]**. We tend to fix critical
-bugs as soon as possible. Feel free to come up with a patch before us though!
+bugs as soon as possible. Feel free to come up with a patch before we do though!
 
 ### Implement Features
 
 Look through the GitHub issues for features. Anything tagged with
 **[improvement]** or **[feature]** is open to whoever wants to implement it.
 
-The issue is unclear or you are not sure what is expected: ask for more
-information by commenting to the issue.
+If the issue is unclear or you are not sure what is expected, ask for more
+information by commenting on the issue.
 
 ### Submit Feedback
 
 Any issue with the **[question]** label is open for feedback, so feel free to
 share your thoughts with us!
 
-The best way to send feedback is to [fill a new
+The best way to send feedback is to [create a new
 issue](https://github.com/TailorDev/monod/issues/new) on GitHub.
 
 If you are proposing a feature:
 
-* Explain how you envision it working. Try to be as detailed as you can;
+* Explain how you envision it working. Try to be as detailed as you can.
 * Try to keep the scope as narrow as possible. This will help make it easier to
-  implement;
+  implement.
```

Comment:
Yeah, I went back and forth on this a few times. The final algorithm I came up with (after a bit of googling grammer rules) was:

* Add a period at the end of full sentences
* Do not add a period at the end of sentence fragments
* Only add a semicolon at the end of sentence fragments if you are starting the next item with the word "and"

I broke the rule above because the last item was actually two sentences, so it felt weird to leave off the period. I think it looks fine without it though, so I've made that change.